app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -261,4 +264,29 @@ public void markMessageAsRead(Message message) {
             unreadMessageHandler.sendEmptyMessageDelayed(0, 2000);
         }
     }
+    public void MuteTopic(int streamId, String subject) {
+        System.out.println((streamId + "-" + subject));
+        muted_topics.add(streamId + "-" + subject);
+        SharedPreferences.Editor editor = settings.edit();
+        editor.putStringSet("muted_topics", muted_topics);
+        editor.commit();        
+    }
+
+    public void UnMuteTopic(int streamId, String subject) {
+        muted_topics.remove(streamId + "-" + subject);
```

Comment:
It seems like adding a "-" is a generic operation that you're doing every time you put/fetch a streamID & subject to the Shared Preferences.  It's a good idea to break this out into a function and use the same logic every place that this operation is required.